src/depool.rs

```diff
@@ -699,3 +766,162 @@ async fn encode_transfer_stake(dest: &str, amount: u64) -> Result<String, String
         "amount": amount
     })).await
 }
+
+async fn call_contract(
```

Comment:
Why we cannot use `send_with_body` and then if flag `answer_is_expected` is true to wait for an answer?

Reply:
As I remember there was an error when I created client more than once and It was not very comfortable to return  it from send_with_body